inc/inventory/request.class.php

```diff
@@ -473,4 +603,49 @@ public function testRules(): Request {
       $this->test_rules = true;
       return $this;
    }
+
+    /**
+     * Accepted encodings
+     *
+     * @return string[]
+     */
+   public function acceptedEncodings(): array {
+      $encodings = [
+         'gzip',
+         'deflate'
+      ];
+
+      if (!function_exists('brotli_uncompress')) {
+         $encodings[] = 'br';
+      }
+
+      return $encodings;
+   }
```

Comment:
I don't see where this is used actually and if glpi returns a convenient error like 415, as said upper, this is probably not necessary.

Reply:
It's not yet used; but this will be used to add tha `Accept` header listed in COMMON specs